shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java

```diff
@@ -156,4 +181,75 @@ public void onEngineWillDestroy() {
   /* package */ FlutterEngine createEngine(Context context) {
     return new FlutterEngine(context);
   }
+
+  /** Options that control how a FlutterEngine should be created. */
+  public static class Options {
+    private Context context;
+    private DartEntrypoint dartEntrypoint;
+    private String initialRoute;
+    private List<String> dartEntrypointArgs;
```

Comment:
Nullable annotations would be helpful here too.

Reply:
Done